Calendar fixes

voidus opened the pull request:
http://jira.opensource.mayflower.de/jira/browse/PHPROJEKT-221?jwupdated=83983&page=com.atlassian.jira.plugin.system.issuetabpanels:all-tabpanel#issue-tabs and small other things

Commit:
Send notifications only once

Instead of having a notify method in Calendar2_Models_Calendar2, overwrite
getNotification to return our custom notification class. This matches default
PHProjekt behaviour.

Commit:
fix not working id field

the id field did not work, we now use the default method (templateRenderer)
which works.
the missing id caused the moduledesigner to delete the old table column and
create a new one, which led to data loss.

dsp committed:
Merge pull request #70 from exi/PHPROJEKT-279

fix not working id field

Commit:
make calendar month view fill the whole screen

The month view now stretches over the whole screen to utilize the extra space on
lange displays

Commit:
Merge branch 'PHPROJEKT-206' into next

Commit:
fix drag drop bug in calendar

after moving an event, there was a javascript error when trying to open
the moved event right away.
this was due to unupdated internal event data that was needed for the query.

exi commented on phprojekt/application/Calendar2/Models/Calendar2.php:
```diff
@@ -1061,6 +1065,18 @@ class Calendar2_Models_Calendar2 extends Phprojekt_Item_Abstract
+
+    /**
+
+    /**
```
multiple opening comments?

Commit:
clear up getModule function of the pagemanager.

we dont need those extra checks, just check for the module or use the error value (null)

exi committed:
Merge pull request #71 from exi/pagemanagerGetModuleFix

clear up getModule function of the pagemanager.

exi committed:
Merge pull request #61 from voidus/fixCalNotifications

Send notifications only once

Commit:
Merge branch 'calendarFixes' into next

* calendarFixes:
  Fix double opening comment
  fixed typo in notifications
  In saveSingleEvent, call save() if the event has no occurrence (again).
  Added protected _originalData to ActiveRecord_Abstract

Conflicts:
	phprojekt/application/Calendar2/Models/Calendar2.php
